content/en/docs/concepts/storage/volumes.md

```diff
@@ -1235,6 +1235,40 @@ Kubernetes component using the following feature gate flags:
 Learn how to
 [setup your PV/PVC with raw block volume support](/docs/concepts/storage/persistent-volumes/#raw-block-volume-support).
 
+#### CSI inline volume support
+
+{{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+
+This feature is to enable support for CSI inline ephemeral volumes to be embedded
```

Comment:
How about something like this?

This feature requires the `CSIInlineVolume` feature gate to be enabled.

This feature allows CSI volumes to be directly specified in the Pod specification instead of through a PersistentVolume.  Volumes specified in this way are ephemeral and do not persist across Pod restarts.  

Here is an example Pod specification:
...

CSI ephemeral volumes are only supported by a subset of CSI drivers.  For a list of CSI drivers please see ....

Reply:
That's pretty good. Thanks. Will update.